### PR TITLE
docs(docs-infra): correct spelling pratices to practices, seperate to s…

### PR DESCRIPTION
**Pull request by MeAkib**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Correct some spellings

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -114,7 +114,7 @@
     "picture": "antonio-cardenas.jpg",
     "twitter": "yeoudev",
     "website": "https://yeou.dev",
-    "bio": "Antonio is a senior Angular developer from Honduras, focus on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Love to share knowledge Typescript Coffe and Cats 🐱.",
+    "bio": "Antonio is a senior Angular developer from Honduras, focus on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Love to share knowledge Typescript Coffee and Cats 🐱.",
```
Comment (ernest-rudnicki) [inline aio/content/marketing/contributors.json:117]: I just noticed a bit weird grammar it this sentence. I think it should be more like this:
```suggestion
    "bio": "Antonio is a senior Angular developer from Honduras, who focuses on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Loves to share knowledge Typescript Coffee and Cats 🐱.",
```

What do you think?

Comment (akib1997) [reply aio/content/marketing/contributors.json:117]: Yes, it can be improved.

Comment (akib1997) [reply aio/content/marketing/contributors.json]: https://github.com/angular/angular/pull/51527/commits/7f4127e36512086bdd827c696de3c3fbfac62573

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -114,7 +114,7 @@
     "picture": "antonio-cardenas.jpg",
     "twitter": "yeoudev",
     "website": "https://yeou.dev",
-    "bio": "Antonio is a senior Angular developer from Honduras, focus on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Love to share knowledge Typescript Coffe and Cats 🐱.",
+    "bio": "Antonio is a senior Angular developer from Honduras, who focuses on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Loves to share knowledge Typescript Coffee and Cats 🐱.",
```
Comment (twerske) [inline aio/content/marketing/contributors.json:117]: This was written by the GDE themselves and I prefer not to edit their bio on their behalf. Please remove from the PR thank you!

**Review by twerske — CHANGES_REQUESTED** [review]
One comment and a lint fix needed! Thank you!

File: aio/content/marketing/contributors.json (hunk 2 above)
Comment (akib1997) [reply aio/content/marketing/contributors.json]: https://github.com/angular/angular/pull/51527/commits/4118afc1cd5002585aea11739ac00f41575ef8f0

**Review by MeAkib — COMMENTED** [review]
I did this by keeping the bio as it was.

**Review by twerske — APPROVED** [review]

**Comment by twerske** [comment]
Please resolve merge conflicts then we can merge

**Comment by JeanMeche** [comment]
In absence of the required changes, I'll close this PR. feel free to reopen a new one.

**Pull request closed without merging**